src/axom/inlet/Table.hpp

```diff
@@ -730,10 +875,116 @@ class Table : public SchemaCreator, public std::enable_shared_from_this<Table>
   bool m_docEnabled;
   std::unordered_map<std::string, std::shared_ptr<Table>> m_tableChildren;
   std::unordered_map<std::string, std::shared_ptr<Field>> m_fieldChildren;
-  std::function<bool()> m_verifier;
+  std::function<bool(Proxy&)> m_verifier;
+};
+
+// To-be-defined template specializations
```

Comment:
Cleanup: Possibly for a separate PR, but I wonder if some of these helper routines should be moved to an implementation/detail file.